
// protoc --cpp_out=. .\actutltype_rpc.proto
// protoc --grpc_out=. --plugin=protoc-gen-grpc="C:\src\vcpkg\installed\x64-windows\tools\grpc\grpc_cpp_plugin.exe" .\actutltype_rpc.proto
// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";



// The greeting service definition.
service Device {
  rpc GetDevice (GetDeviceRequest) returns (GetDeviceReply) {}
  rpc SetDevice (SetDeviceRequest) returns (SetDeviceReply) {}
}

// The request message containing the user's name.
message GetDeviceRequest {
  string Device = 1;
}

// The response message containing the greetings
message GetDeviceReply {
  int32 message = 1;
}


message SetDeviceRequest {
  string Device = 1;
  int32 Data = 2;
}

// The response message containing the greetings
message SetDeviceReply {
  int32 message = 1;
}